### ISSUE TITLE
search: broken recovery flow

### ISSUE THREAD

**Issue by pmwkaa** [tarantool/tarantool#1101]
```
2015-10-19 14:27:33.192 [28471] main/103/iproto crowstorage_tarantool.cpp:418 E> box_replace() failed
2015-10-19 14:27:33.192 [28471] main/103/iproto crowstorage_tarantool.cpp:719 E> ReplaceKeyValue() failed (/home/a.levshin/rmake/
 2015-10-19 14:27:33.192 [28471] main/103/iproto crowstorage_tarantool.cpp:795 E> UnpackAndSaveTZData() failed; /home/a.levshi
 2015-10-19 14:27:33.202 [28471] main/103/iproto crowstorage_tarantool.cpp:418 E> box_replace() failed
 2015-10-19 14:27:33.202 [28471] main/103/iproto crowstorage_tarantool.cpp:719 E> ReplaceKeyValue() failed (/home/a.levshin/rm
 2015-10-19 14:27:33.202 [28471] main/103/iproto crowstorage_tarantool.cpp:795 E> UnpackAndSaveTZData() failed; /home/a.levshi
```

```
(gdb) backtrace
#0 __cxxabiv1::__cxa_throw (obj=0x93ea40, tinfo=0x618200 <typeinfo for ClientError*>, dest=0x0) at ../../../../libstdc++-v3/libs
#1 0x000000000044e615 in space_cache_find (id=<optimized out>) at /mnt/24/tarantool-instance-bench/tarantool/src/box/schema.h:97
#2 process_rw (request=request@entry=0x7fffee84f2d0, result=result@entry=0x7fffee84f388) at /mnt/24/tarantool-instance-bench/tar

(gdb) frame 1
#1 0x000000000044e615 in space_cache_find (id=<optimized out>) at /mnt/24/tarantool-instance-bench/tarantool/src/box/schema.h:97
97 tnt_raise(ClientError, ER_NO_SUCH_SPACE, int2str(id));
```

Seems to be box.cfg{} yields and tries to process incoming calls before the space is being created (empty database).

**Comment by rtsisyk** [comment]
Confirmed.

**Comment by rtsisyk** [comment]
Fixed in 1.6.7-257-gba82c62
